robot/markers.py

```diff
@@ -1,30 +1,16 @@
 import math
+from collections import namedtuple
+from typing import List, NewType, Tuple
 
 from robot.game_specific import TOKEN, WALL
 
+TMetres = NewType('TMetres', float)
+TDegrees = NewType('TDegrees', float)
+TRadians = NewType('TRadians', float)
+TPixel = NewType('TPixel', float)
```

Comment:
Is it really intentional that `Pixel` is `float` rather than `int`? (Do we really have pixel values which are fractional?)

Reply:
hmm, youre right, it seems unlikely to need fractional pixels. Is this addressed in #59 ?

Reply:
no